benchmarks/orc/sample_zillow.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python3
+# (c) L.Spiegelberg 2019
+
```

Comment:
For the benchmark scripts - are there any instructions on how to run them/reproduce them? If not, please add a README.md file detailing step-by-step what todo in order to get the benchmarks running.